src/main/java/frc/robot/commands/SensitiveDriverControl.java

```diff
@@ -0,0 +1,54 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.*;
+
+/**
+ * Triggers a modifier which limits the speed the robot can turn and only allows it to turn in place
+ */
+public class SensitiveDriverControl extends CommandBase {
+    
+    //get the drivebase
+    private DriveBase driveBase;
+    private Joystick j;
+
+    public SensitiveDriverControl(Joystick j) {
+        //set the drivebase
+        addRequirements(DriveBase.getInstance());
+        this.driveBase = DriveBase.getInstance();
```

Comment:
nitpick: I would put this line first and then assign it to the addRequirements, e.g.
```
this.driveBase = DriveBase.getInstance();
addRequirements(this.driveBase);
```